### ISSUE TITLE
I already upgrade gradio version 4.14.0 to 4.29.0 but the warning text doesn't disappear.

### ISSUE THREAD

**Issue by lee-jin-01** [jhj0517/Whisper-WebUI#146]
**Which OS are you using?**
 - OS: [e.g. iOS or Windows.. If you are using Google Colab, just Colab.]
Win 11

![gradio ver](https://github.com/jhj0517/Whisper-WebUI/assets/65181724/c8f75b42-dbbd-45c3-a683-e92b106b1318)
![web ui](https://github.com/jhj0517/Whisper-WebUI/assets/65181724/86d544ef-a95d-45dd-8895-f610f006d9a4)

The program operation is normal and there are no other problems, but is it okay?
I'm sorry if I hurt your feelings because it's a basic question. 

btw the program you made is very useful and convenient, so I'm using it well and loving it. thanks for your hard work!

**Comment by jhj0517** [comment]
Hi, please feel free to open the issue if you have any problem!

I think you installed gradio on your PC not in the `venv` folder.
This web ui has the `venv` folder which contains all the dependencies of the web ui. 

You should
1. activate venv
```
./venv/Scripts/activate
```
2. upgrade gradio version
```
pip install gradio==4.29.0 --upgrade
```

**Comment by jhj0517** [comment]
This is fixed in #149, the message will not appear in the latest version of the web ui.
Please feel free to re-open it if the problem persists!

**Comment by lee-jin-01** [comment]
Thank you for your kind explanation, and I've worked it out.
Thank you very much! have a nice day!
